rskj-core/src/test/java/co/rsk/peg/federation/NonStandardErpFederationsTest.java

```diff
@@ -258,6 +258,45 @@ void testEquals_same() {
         assertEquals(nonStandardErpFederation, otherFederation);
     }
 
+    @Test
+    void values_from_erpFederationArgs_equal_values_from_nonStandardErpFederation() {
+        ErpFederationArgs erpFederationArgs = nonStandardErpFederation.getErpArgs();
```

Comment:
What do you think about adding an `equals` function to FederationArgs? I think that would simplify these tests and it's always a good idea for java object to have an equals function

Reply:
good one! thanks

Reply:
Since we are implementing equals and hashcode properly, what about implementing the `toString` method. Not a big deal but it would be useful in case we want to print `federationArgs` object.

Reply:
good idea as well :)
thanks guys